stripe2qbo/tests/test_transactions.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from typing import Annotated
+
+from sqlalchemy.orm import Session
+from fastapi import Depends
+
+from stripe2qbo.api.dependencies import get_current_user, get_db
+from stripe2qbo.api.routers.transaction_router import (
+    get_all_transactions,
+    get_transaction_by_id,
+)
+from stripe2qbo.db.models import TransactionSync, User, QBOToken
+
+pytest_plugins = ("pytest_asyncio",)
+
+
+@pytest.fixture
+def create_user():
+    test_qbo_token = QBOToken(
+        realm_id="123",
+        access_token="123",
+        refresh_token="123",
+        expires_at="123",
+        refresh_token_expires_at="123",
+        user_id=1,
+    )
+    test_qbo_token.save()
```

Comment:
I'm not sure this is valid syntax for SQL Alchemy

Could be wrong. I know this is how I did it for other models:
```
db.add(test_qbo_token)
db.commit()
```